integration_test/cases/preload.exs

```diff
@@ -708,6 +709,37 @@ defmodule Ecto.Integration.PreloadTest do
     assert %User{id: ^uid1} = item.user
   end
 
+  test "preload belongs_to in embedded_schema from container schema" do
+    # Order embeds_many items
+    # Items belong to Users
+    # Preload Users from an Order schema
+
+    %User{id: uid1} = TestRepo.insert!(%User{name: "1"})
+    %User{id: uid2} = TestRepo.insert!(%User{name: "2"})
+    %User{id: uid3} = TestRepo.insert!(%User{name: "3"})
+    item1 = %Item{id: 1, user_id: uid1}
+    item2 = %Item{id: 2, user_id: uid2}
+    item3 = %Item{id: 3, user_id: uid3}
+    order = %Order{items: [item1, item3, item2]}
+    %Order{id: oid1} = TestRepo.insert!(order)
```

Comment:
```suggestion
    %Order{} = TestRepo.insert!(order)
```
`oid1` is causing a warning because it's not being used anywhere:

>variable "oid1" is unused (if the variable is not meant to be used, prefix it with an underscore)